test/bootstrap.js

```diff
@@ -9,13 +9,6 @@ import { Adapter } from 'enzyme-adapter-preact-pure';
 
 configure({ adapter: new Adapter() });
 
-// Make all the icons that are available for use with `SvgIcon` in the actual
-// app available in the tests. This enables validation of icon names passed to
-// `SvgIcon`.
-import testIcons from '../src/icons';
-import { registerIcons } from '../src/components/SvgIcon';
-registerIcons(testIcons);
-
```

Comment:
Components should be registering their needed icons, so let's get rid of these training wheels.